tests/integration/connect_setup.go

```diff
@@ -413,16 +442,31 @@ func (s *ConnectIntegrationSuite) AddCurrencyPairs(chain *cosmos.CosmosChain, us
 	// get an rpc endpoint for the chain
 	client := chain.Nodes()[0].Client
 
+	ctx := context.Background()
+
 	// broadcast the tx
-	resp, err := client.BroadcastTxCommit(context.Background(), tx)
+	txResp, err := client.BroadcastTxCommit(ctx, tx)
 	if err != nil {
 		return err
 	}
 
-	if resp.TxResult.Code != abcitypes.CodeTypeOK {
-		return fmt.Errorf(resp.TxResult.Log)
+	if txResp.TxResult.Code != abcitypes.CodeTypeOK {
```

Comment:
Do proposals pass immediately with interchaintest?

Reply:
No, but the result of a proposal or submitted Tx will be TypeOK if it landed successfully